packages/flutter_tools/lib/src/project.dart

```diff
@@ -822,6 +822,28 @@ class AndroidProject extends FlutterProjectPlatform {
   /// True if the Flutter project is using the AndroidX support library.
   bool get usesAndroidX => parent.usesAndroidX;
 
+  /// Returns true if the current version of the Gradle plugin is supported.
+  bool isSupportedVersion() {
```

Comment:
Idiomatic Dart would make this a getter rather than a no-arg method. I'd also suggest caching the result if this will be called multiple times and the results won't/shouldn't change.

Reply:
Done

Reply:
It looks like the callsites need to be updated.

Reply:
slowly editing through the github UI :)